### PR TITLE
Image for Regression test for #74839

### PR DESCRIPTION
**Pull request by anfernee**
Followup for #74840

```release-note
```

### CODE REVIEW

**Comment by anfernee** [comment]
@MrHohn this adds the test image. tried `make all WHAT=issue-74839`

File: test/images/issue-74839/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM BASEIMAGE
```
Comment (MrHohn) [inline test/images/issue-74839/Dockerfile]: Seems like we will need boilerplate in dockerfile as well. Ref https://github.com/kubernetes/kubernetes/blob/master/test/images/netexec/Dockerfile

File: test/images/issue-74839/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/issue-74839
```
Comment (MrHohn) [inline test/images/issue-74839/.gitignore]: Should we maybe give this folder a more meaningful name? e.g. invalid-packet-test

File: test/images/issue-74839/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Reproduction of k8s issue #74839
+
```
Comment (MrHohn) [inline test/images/issue-74839/README.md]: Would be great to have a one-line description here, even better if we note down what the test does from a high level :)

**Review by MrHohn — APPROVED** [review]
/approve

File: test/images/issue-74839/.gitignore (hunk 2 above)
Comment (anfernee) [reply test/images/issue-74839/.gitignore]: I was thinking calling it regression-issue-74839 to indicate it's a regression test.

Comment (MrHohn) [reply test/images/issue-74839/.gitignore]: Sure, that sounds better.

Comment (anfernee) [reply test/images/issue-74839/.gitignore]: we can add more details in the README.

File: test/images/regression-issue-74839/BASEIMAGE [hunk 4]
```diff
@@ -0,0 +1,4 @@
+amd64=alpine:3.6
```
Comment (listx) [inline test/images/regression-issue-74839/BASEIMAGE]: Can we avoid alpine and instead try to use [distroless](https://github.com/GoogleContainerTools/distroless)?

Comment (MrHohn) [reply test/images/regression-issue-74839/BASEIMAGE]: Should be doable assuming we don't have special dependency here. Ref https://github.com/kubernetes/dns/pull/294 for a similar change.

**Comment by MrHohn** [comment]
/kind bug

Adding a hold to see if we should use distroless.
/hold

**Pull request merged**